examples/web-wagmi/vite.config.ts

```diff
@@ -11,19 +11,4 @@ export default defineConfig({
     nodePolyfills({ protocolImports: true }),
     visualizer({ open: true }) as PluginOption,
   ],
-
-  build: {
-    target: 'esnext',
-    minify: 'esbuild',
-
-    rollupOptions: {
-      treeshake: {
-        preset: 'recommended',
-      },
-    },
-  },
-
-  define: {
-    'process.env': {},
```

Comment:
@desfero did we check metadata upload via WebBundlr works still? This line and the use of `nodePolyfills` plugin were needed to make `WebBundlr` works on browsers.